src/main/java/net/dv8tion/jda/api/utils/ImageProxy.java

```diff
@@ -52,7 +53,10 @@ public ImageProxy(@Nonnull String url)
      * <br>The size is a best-effort resize from Discord, with recommended size values as powers of 2 such as 1024 or 512.
      *
      * @param  size
-     *         The size of the image
+     *         The width of the image
```

Comment:
Why did this change?

Reply:
It seems the requested size matches the width